jenkins/bootstrap.py

```diff
@@ -967,6 +974,24 @@ def parse_args(arguments=None):
         '--clean',
         action='store_true',
         help='Clean the git repo before running tests.')
+    # Use "_no_-magic-env" because the default (current functionality)
+    # is to have magic environment, so the default value for this flag
+    # (false) needs to match this behavior.
+    parser.add_argument(
+        '--no-magic-env',
```

Comment:
yes thank you for the ideas I disliked the current name for that reason, `raw-env` sgtm